ONSdigital/eq-questionnaire-runner
# Bug fix: Fields not honouring schema defined decimal places

### What is the context of this PR?
Fix bug that changed number of decimal places after submitting and returning to edit percentage field. 

### How to review 
1. Run `test_percentage` schema, enter a decimal number to three decimal places(dp). Submit then click previous to go back and edit number. Verify that the number hasn't been amended to 2dp and still remains at 3dp. 
2. Run functional test percentage_decimal.spec.js and ensure that test passes.

### Checklist
